index.html

```diff
@@ -182,14 +182,18 @@ <h3 id="unacceptablebehaviour">
         <li>Retaliating against anyone who files a complaint that someone has violated this code of conduct.</li>
       </ul>
 
-      <p>The enforcers of this Code should prioritise the safety of individuals within marginalised communities over the comfort of others,
-        and reserve the right not to act on complaints regarding:</p>
+      <p>
+	All complaints will be taken seriously and will recieve a response, 
+	but enforcers of this Code will prioritise the safety of individuals
+	within marginalised communities over the comfort of others,
+        and reserve the right not to take an action on complaints regarding:
```

Comment:
suggest s/take an action/take further action

Reply:
Sounds good to me will do.